src/store/user/userOperations.ts

```diff
@@ -0,0 +1,46 @@
+import { createAsyncThunk, nanoid } from "@reduxjs/toolkit";
```

Comment:
I think better name this file `userThunks.ts`. It's more undetstandable for other.
And really we if you want we can just call files `thunks.ts` and `slice.ts` because we have folder use and all understand that it's user thunks and slice